tests/functions/test_map_to_s3.py

```diff
@@ -3,26 +3,47 @@
 import mock
 import json
 
-TEST_PATH = "www.example.com/content/file.ext"
+TEST_PATH = "/origin/rpms/repo/ver/dir/filename.ext"
 MOCKED_DT = "2020-02-17T15:38:05.864+00:00"
 CONF_PATH = "cdn_lambda/functions/map_to_s3/map_to_s3.json"
 
 
+@pytest.mark.parametrize(
+    "req_uri, real_uri",
```

Comment:
Can we please also have a case where we do NOT do any rewriting?  (e.g. to protect against a bug where matching no aliases at all causes a crash.)

Reply:
Done